.storybook/main.js

```diff
@@ -83,7 +84,15 @@ module.exports = {
       },
     };
 
-    // console.dir(resultConfig, { depth: null });
+    // Addons have to be loaded with babel
+    const addonPaths = registry.addonNames.map((addon) =>
+      fs.realpathSync(registry.packages[addon].modulePath),
+    );
+    resultConfig.module.rules[1].exclude = (input) =>
+      // exclude every input from node_modules except from @plone/volto
+      /node_modules\/(?!(@plone\/volto)\/)/.test(input) &&
+      // If input is in an addon, DON'T exclude it
+      !addonPaths.some((p) => input.includes(p));
```

Comment:
I copied this over from our project, it is also added in the generator (see below), I'm not entirely sure it is needed in the core configuration as well. Needs testing.